NEWS.md

```
@@ -12,20 +12,38 @@ To read all history of releases go to the latest [release notes](https://docs.pg
+## pgRouting 4.0
 
-Code enhancement
 
-* [#3025](https://github.com/pgRouting/pgrouting/issues/3025): Ordering functions standardized
+<details> <summary>pgRouting 4.0.1 Release Notes</summary>
 
+To see all issues & pull requests closed by this release see the
+[#4.0.1](https://github.com/pgRouting/pgrouting/issues?utf8=%E2%9C%93&q=milestone%3A%22Release%204.0.1%22)
 
-## pgRouting 4.0
+**Bug Fixes**
```

markdownlint-cli2 (0.20.0)

[warning] Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)